packages/manager/modules/log-to-customer/src/service.js

```diff
@@ -0,0 +1,41 @@
+export default class LogToCustomerService {
+  /* @ngInject */
+  constructor($http, $q) {
+    this.$http = $http;
+    this.$q = $q;
+  }
+
+  getDataStreams() {
+    return this.$http.get('/dbaas/logs').then(({ data }) => {
```

Comment:
TODO add parameter

Reply:
Same : this should be common to every services